engine/morejson/positive_duration.go

```diff
@@ -0,0 +1,33 @@
+package morejson
+
+import (
+	"encoding/json"
+	"errors"
+	"time"
+)
+
+// PositiveDuration is a wrapper around time.Duration.
+// It restricts its values to positive ones only, and it implements the
+// json.Marshaler and json.Unmarshaler interfaces.
+type PositiveDuration struct {
```

Comment:
I don't think we need a cusom time. You coul either check the value of the `time.Duration` value at runtime, or simply ignore it, since it is only used by `time.Sleeps`, which state in its doc:
> A negative or zero duration causes Sleep to return immediately.

Reply:
Right, didn’t notice that. What should I do?
- scrap this type, set `step.LoopDelay`’s type to `string`, and parse the duration in `AfterRun` (with a warning log if the parsing fails)?
- keep this type but refactor it?

Reply:
What about: `step.LoopDelay` as a `time.Duration`, and validate inside the function `step.ValidAndNormalize` (https://github.com/ovh/utask/blob/master/engine/step/step.go#L512) that the value is not negative ?
That's mean that the error will be triggered at boot time, instead of run time; and get rides of this new type ?